src/ledger/LedgerManagerImpl.h

```diff
@@ -94,7 +94,8 @@ class LedgerManagerImpl : public LedgerManager
                  std::unique_ptr<LedgerCloseMetaFrame> const& ledgerCloseMeta,
                  uint32_t initialLedgerVers);
 
-    void storeCurrentLedger(LedgerHeader const& header, bool storeHeader);
+    void storeCurrentLedger(LedgerHeader const& header, bool storeHeader,
+                            bool checkpoint);
```

Comment:
I misread/misunderstood `checkpoint` as a parameter name and thought it meant "this ledger is a checkpoint boundary" not "this call should store to the checkpoint-in-progress". maybe rename as `alsoAppendToCheckpoint`?